sql/addons_aggregates_v3.sql

```diff
@@ -0,0 +1,63 @@
+WITH
+  extended_addons AS (
+  SELECT
+  submission_date_s3 as submission_date,
+    sample_id,
+    client_id,
+    normalized_channel,
+    SUBSTR(app_version, 1, 2) AS app_version,
+    locale,
+    _addon.element.addon_id AS addon_id,
+    IF(_addon.element.foreign_install,
+      1,
+      0) AS is_foreign_install,
+    IF(_addon.element.is_system,
+      1,
+      0) AS is_system,
+    IF(_addon.element.is_web_extension,
+      1,
+      0) AS is_web_extension,
+    _addon.element.install_day AS install_day,
+    IF(_addon.element.addon_id IS NOT NULL
+      AND NOT _addon.element.is_system
+      AND NOT _addon.element.foreign_install
+      AND NOT _addon.element.addon_id LIKE '%mozilla%'
+      AND NOT _addon.element.addon_id LIKE '%cliqz%'
+      AND NOT _addon.element.addon_id LIKE '%@unified-urlbar%',
+      1,
+      0) AS is_self_install,
+    IF(_addon.element.addon_id LIKE "%@shield.mozilla%",
+      1,
+      0) AS is_shield_addon,
+    profile_creation_date
+  FROM
+    telemetry.main_summary_v4,
+    UNNEST(active_addons.list) AS _addon
+  WHERE
+    submission_date_s3 = @submission_date
+    AND client_id IS NOT NULL)
+SELECT
+  DISTINCT submission_date,
+  sample_id,
+  client_id,
+  normalized_channel,
+  app_version,
+  locale,
+  SUM(is_self_install) AS n_self_installed_addons,
+  SUM(is_shield_addon) AS n_shield_addons,
+  SUM(is_foreign_install) AS n_foreign_installed_addons,
+  SUM(is_system) AS n_system_addons,
+  SUM(is_web_extension) AS n_web_extensions,
```

Comment:
Instead of using `SUM` like the python version, I think we should use `COUNTIF` and leave the addon columns boolean (see windowing comment for examples)

Reply:
Thanks, I assumed there was something like `COUNTIF` but didn't find it yet. :)